### PR TITLE
Form assets media as a dynamic property

### PR DESCRIPTION
**Pull request by pincoin**
Hello,

I changed `settings` not to call `static` functions because it causes some problems when using `ManifestStaticFilesStorage` such as cached storage.

https://docs.djangoproject.com/ko/1.11/topics/forms/media/#media-as-a-dynamic-property

Thank you.

### CODE REVIEW

**Comment by pincoin** [comment]
My PR was failed because of Travis hang. You may check that my commit works fine.

https://travis-ci.org/summernote/django-summernote

Thank you.

File: djs_playground/settings.py [hunk 1]
```diff
@@ -117,6 +117,8 @@
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 
 STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'djs_playground/static/')
+# STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'
```
Comment (lqez) [inline djs_playground/settings.py]: Could you remove the commented-out line at 121L

Comment (pincoin) [reply djs_playground/settings.py]: I intentionally put it there in order to imply our playground supports storage settings.

However, I will send PR after removing out. Thank you.

Comment (lqez) [reply djs_playground/settings.py]: Thanks! And I saw your PR was successfully built.

Comment (lqez) [reply djs_playground/settings.py]: Please let me handle this. Thanks.

**Pull request merged**